src/Lykke.Service.ExchangeDataStore/Lykke.Service.ExchangeDataStore/appsettings.json

```diff
@@ -0,0 +1,15 @@
+﻿{
+  // NOTE: Please, modify this settings only to reflect AppSettings structure.
+  // If you want to use local settings, add appsettings.Development.json file, which not version controlled
+  "ExchangeDataStoreService": {
```

Comment:
Change the settings file according the AppSettings class

Reply:
Done.